{"input_dir": "/root/input", "input_files": ["/root/input/DXm.2.25.119403010146292111130118207990186551199.dcm"], "output_dir": "/root/output/input", "_metadata": null, "study_metadata": {"PatientName": "", "PatientID": "DHJ5JE9QXAV0", "PatientFirstName": "", "PatientLastName": "", "PatientBirthDate": "", "PatientSex": "", "AccessionNumber": "DHMO4N9W6KD1", "StudyInstanceUID": "2.25.100000201839274263174475898571171534853", "StudyDate": ""}, "_checker_validation_passed": false, "_failed_acceptance_criteria_list": [], "results_raw": null, "results": null, "results_file_name": "results_full.json", "reports_expected": [], "reports_generated": {}, "output_send_results": {}, "preprocessed": false, "predicted": false, "outputs_sent": false, "_error_code": 1, "_error_messages": ["Traceback (most recent call last):\n  File \"/root/centaur/centaur_deploy/deploys/base_deployer.py\", line 318, in run_study\n    accepted = self.engine.preprocess(input_dir=self.config[Config.MODULE_IO, 'input_dir'])\n  File \"/root/centaur/centaur_engine/engine.py\", line 87, in preprocess\n    passed = self.preprocessor.preprocess(self.file_list)\n  File \"/root/centaur/centaur_engine/preprocessor.py\", line 214, in preprocess\n    np_data = set_pixel_array((dr, len(self.metadata), save_to_ram))\n  File \"/root/centaur/centaur_engine/helpers/helper_preprocessor.py\", line 125, in set_pixel_array\n    pixel_array = dr.get_ds().pixel_array\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 835, in __getattr__\n    return object.__getattribute__(self, name)\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 1716, in pixel_array\n    self.convert_pixel_data()\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 1435, in convert_pixel_data\n    self._convert_pixel_data_without_handler()\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 1545, in _convert_pixel_data_without_handler\n    raise last_exception\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 1525, in _convert_pixel_data_without_handler\n    self._do_pixel_data_conversion(handler)\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 1554, in _do_pixel_data_conversion\n    arr = handler.get_pixeldata(self)\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/pixel_data_handlers/gdcm_handler.py\", line 237, in get_pixeldata\n    gdcm_data_element = create_data_element(ds)\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/pixel_data_handlers/gdcm_handler.py\", line 124, in create_data_element\n    pydicom.encaps.defragment_data(ds.PixelData)\n  File \"/opt/conda/lib/python3.7/site-packages/pydicom/dataset.py\", line 835, in __getattr__\n    return object.__getattribute__(self, name)\nAttributeError: 'FileDataset' object has no attribute 'PixelData'\n"], "studies_db_id": null, "uid": null}